common/lifestyle_perks/00_martial_1_strategy_tree_perks.txt

```diff
@@ -44,8 +44,18 @@ parthian_tactics_perk = {
 	character_modifier = {
```

Comment:
Errors.
```
[19:45:27][localization_reader.cpp:208]: Expected quotation mark (") at end of line 4 in localization/english/wc_game_concepts_l_english.yml
[19:45:27][gameconcepts.cpp:80]: Game concept 'giant': Missing localization key 'game_concept_giant'
[19:45:37][lexer.cpp:332]: File 'common/culture/innovations/wc_regional_innovations.txt' should be in utf8-bom encoding (will try to use it anyways)
```

Reply:
I tried fixing this before, I think it has to do with the giants you made in MAA file

Every other error is fixed

Reply:
> I tried fixing this before, I think it has to do with the giants you made in MAA file
> 
> Every other error is fixed

You made this game concept.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105872939-654ba200-6014-11eb-9528-92f9ff12db61.png)

</details>